src/distill.c

```diff
@@ -91,9 +91,9 @@ void program_distill(void) {
                         0.018487, 4.1145 * pow(10, -6));
 
       switch (mode_change) {
-      // heanting control part (PID)
+      // heating control part (PID)
       case 0:
-        if (switched == 0) {
+        if (switched == HEATER_OFF) {
           switch_ON = heater_switch(PI_value);
           if (switch_ON == 1) {
```

Comment:
you're declaring `switch_ON` to be of type `HeaterStatus`, please use the same type when checking for it